modules/rosa-configure-privatelink-architecture.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * cloud_infrastructure_access/rosa-aws-privatelink.adoc
+
+[id="rosa-configure-privatelink-architecture.adoc_{context}"]
+= PrivateLink architecture
+
+The Red Hat managed infrastructure responsible for creating PrivateLink clusters are hosted on completely private subnets. The only connection made between Red Hat and the customer provided infrastructure is through PrivateLink VPC Endpoints. 
+
+The following diagram shows an example of a multiple Availability Zone (Multi-AZ) PrivateLink cluster deployed only on private subnets. 
+
+.Multi-AZ PrivateLink cluster deployed on private subnets
+image::rosa-privatelink-architecture.png[Multi-AZ PrivateLink cluster deployed on private subnets]
+
+AWS provides multiple reference architectures. Here are three examples:
+
+* VPC with a private subnet and AWS Site-to-Site VPN access.
++
+This configuration enables you to extend your network into the cloud without exposing your network to the internet.
++
+It contains a virtual private cloud (VPC) with a single private subnet and a virtual private gateway to enable communication with your network over an IPsec VPN tunnel. There is no Internet gateway to enable communication over the Internet. 
++
+For more information, see link:https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Scenario4.html[VPC with a private subnet only and AWS Site-to-Site VPN access] in the AWS documentation.
+
+* VPC with public and private subnets (NAT)
++
+This configuration enables you to isolate your network so that the public subnet is reachable from the internet but the private subnet is not.
++
+Only the public subnet can send outbound traffic directly to the Internet. The private subnet can access the internet by using a network address translation (NAT) gateway that resides in the public subnet. This allows database servers to connect to the Internet for software updates using the NAT gateway, but does not allow connections to be made directly from the Internet to the database servers.
++
+For more information, see link:https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Scenario2.html[VPC with public and private subnets (NAT)] in the AWS documentation.
+
+* VPC with public and private subnets and AWS Site-to-Site VPN access
++
+This configuration enables you to extend your network into the cloud and also directly access the internet from your VPC
```

Comment:
Suggest adding a period to the end of this sentence.

s/and also directly access/and to directly access